ql/src/test/queries/clientpositive/udf_to_unix_timestamp.q

```diff
@@ -18,8 +18,8 @@ SELECT
 FROM oneline_n0;
 
 SELECT
-  '2009 Mar 20 11:30:01 am',
-  to_unix_timestamp('2009 Mar 20 11:30:01 am', 'yyyy MMM dd h:mm:ss a')
+  '2009 Mar 20 11:30:01 AM',
```

Comment:
Why is this change required? Lower case doesn't work?

Reply:
simpleDateFromat and DateTimeformatter both used locale(https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html) to parse alphabet. By default system local in picked to in both the case. simpleDateFromat use Locale.getDefault(Locale.Category.FORMAT)) which is list of locale due to "AM" and "am" but not "aM" or "Am". Where as DateTimeformatter used only system default locale also DateTimeformatter by default in case Sensitive as well. If supports only "AM"/"am" which is not in our control. I Do align the point the date value should be case insensitive but I couldn't able to find any DateTimeFromatter implementation with in hive code. But still in order to make unix_timestamp backword compatible we need to make DateTimeformatter case insensitive.